libsplinter/src/admin/service/mailbox/mod.rs

```diff
@@ -12,16 +12,25 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//! Data store for admin events.
+//!
+//! The [`Mailbox`] provides storage for `AdminServiceEvent`s.
+//!
+//! [`Mailbox`]: struct.Mailbox.html
+
+#[cfg(any(feature = "admin-mailbox-postgres", feature = "sqlite"))]
+mod diesel;
```

Comment:
This should not be guarded by sqlite like this. This makes the code stable if sqlite is turned on. It instead should be guarded by "admin-mailbox-diesel" or something like that.